charts/px-backup/templates/px-backup/pxcentral-etcd.yaml

```diff
@@ -0,0 +1,364 @@
+    ETCDCTL_ENDPOINTS="http://px-backup-etcd-0.px-backup-etcd-headless.{{ .Values.namespace }}.svc.cluster.local:2379,http://px-backup-etcd-1.px-backup-etcd-headless.{{ .Values.namespace }}.svc.cluster.local:2379,http://px-backup-etcd-2.px-backup-etcd-headless.{{ .Values.namespace }}.svc.cluster.local:2379,"
+    # Remove the last comma "," introduced in the string
+    export ETCDCTL_ENDPOINTS="$(sed 's/,/ /g' <<< $ETCDCTL_ENDPOINTS | awk '{$1=$1};1' | sed 's/ /,/g')"
+
+    etcdctl $AUTH_OPTIONS member remove --debug=true "$(cat "$ETCD_DATA_DIR/member_id")" > "$(dirname "$ETCD_DATA_DIR")/member_removal.log" 2>&1
+  probes.sh: |-
+    #!/bin/bash
+
+    set -o errexit
+    set -o pipefail
+    set -o nounset
+
+    # Debug section
+    exec 3>&1
+    exec 4>&2
+
+    if [ "${BITNAMI_DEBUG}" = true ]; then
+      echo "==> Bash debug is on"
+    else
+      echo "==> Bash debug is off"
+      exec 1>/dev/null
+      exec 2>/dev/null
+    fi
+
+    # Constants
+    AUTH_OPTIONS=""
+
+    echo "==> [DEBUG] Probing etcd cluster"
+    echo "==> [DEBUG] Probe command: \"etcdctl $AUTH_OPTIONS endpoint health\""
+    etcdctl $AUTH_OPTIONS endpoint health
+---
+apiVersion: v1
+kind: Service
+metadata:
+  name: px-backup-etcd-headless
+  namespace: {{ .Values.namespace }}
+  labels:
+    app.kubernetes.io/name: etcd
+    app.kubernetes.io/instance: px-backup-etcd
+{{- include "px-backup.labels" . | nindent 4 }}
+  annotations:
+    service.alpha.kubernetes.io/tolerate-unready-endpoints: "true"
+spec:
+  type: ClusterIP
+  clusterIP: None
+  ports:
+    - name: client
+      port: 2379
+      targetPort: client
+    - name: peer
+      port: 2380
+      targetPort: peer
+  selector:
+    app.kubernetes.io/name: etcd
+    app.kubernetes.io/instance: px-backup-etcd
+---
+apiVersion: v1
+kind: Service
+metadata:
+  name: px-backup-etcd
+  namespace: {{ .Values.namespace }}
+  labels:
+    app.kubernetes.io/name: etcd
+    app.kubernetes.io/instance: px-backup-etcd
+{{- include "px-backup.labels" . | nindent 4 }}
+spec:
+  type: ClusterIP
+  ports:
+    - name: client
+      port: 2379
+      targetPort: client
+      nodePort: null
+    - name: peer
+      port: 2380
+      targetPort: peer
+      nodePort: null
+  selector:
+    app.kubernetes.io/name: etcd
+    app.kubernetes.io/instance: px-backup-etcd
+---
+apiVersion: apps/v1
+kind: StatefulSet
+metadata:
+  name: px-backup-etcd
+  namespace: {{ .Values.namespace }} 
+  labels:
+    app.kubernetes.io/name: etcd
+    app.kubernetes.io/instance: px-backup-etcd
+{{- include "px-backup.labels" . | nindent 4 }}
+spec:
+  selector:
+    matchLabels:
+      app.kubernetes.io/name: etcd
+      app.kubernetes.io/instance: px-backup-etcd
+  serviceName: px-backup-etcd-headless
+  podManagementPolicy: Parallel
+  replicas: 3
+  updateStrategy:
+    type: RollingUpdate
+  template:
+    metadata:
+      labels:
+        app.kubernetes.io/name: etcd
+        app.kubernetes.io/instance: px-backup-etcd
```

Comment:
Don't need these. Should be `px-backup.labels` instead

Reply:
Changed it to px-backup.labels